tools/bapbundle.ml

```diff
@@ -53,13 +53,25 @@ let named_resource r = match String.split ~on:'=' r with
   | [name;file] -> Some name, file
   | _ -> None,r
 
-let open_bundle () = match target.contents with
+module Plugin_bundle = struct
```

Comment:
This is kind of strange choice of naming, as well as a strange module.  You named a module as a `Plugin_bundle` so a person who will later read/modify this code, will assume that the module is responsible for the plugin's bundle implementation. However, it is only responsible for handling filenames of plugins. Moreover, some functionality of the module (the `rename_if_plugin` function) leaked out (that's why you were in need to use local open).
 
If you really think, that we need any modularization here, then I would suggest to name it `Name`.  
The `rename_if_plugin` function should be moved inside, and renamed to `normalized`. Speaking about using the  `rename`  verb in function names: this name sounds like an action, that has a side effect. A casual reader, will think that function `rename` will actually rename files on a filesystem. In general, try not to use verbs in names of pure functions. Stick to nouns, adjectives and participles. If a function performs non-desctructive transformation (aka functional update), use either `with_*` form, or past participle, e.g., `Student.with_name student newname`, or `Student.renamed student newname`.